src/cdk/v2/destinations/fullstory/procWorkflow.yaml

```diff
@@ -76,7 +76,7 @@ steps:
         "use_most_recent": .message.properties.useMostRecent,
       };
       $.context.payload.user = {
-        "id": .message.properties.userId ?? .message.userId,
+        "uid": .message.properties.userId ?? .message.userId,
```

Comment:
Is this breaking change ?

Reply:
this is a breaking change. cc @sandeepdsvs